app-connector-client/package.json

```diff
@@ -19,7 +19,6 @@
     "check_api": "^1.2.2",
     "commander": "^2.20.0",
     "cors": "^2.8.5",
```

Comment:
do we still need cors here?

Reply:
yes I need it in the app-connector-client

Reply:
ok